src/components/figures/chart/chartJs/chartjs_show_values_plugin.ts

```diff
@@ -0,0 +1,69 @@
+import { ChartType, Plugin } from "chart.js";
+import { chartFontColor } from "../../../../helpers/figures/charts/chart_common";
+
+interface ChartShowValuesPluginOptions {
+  humanize: boolean;
```

Comment:
It may not be the correct term, but if we want something else we also have to change the scorecard design panel (not a problem for me, just saying it should be discussed with François)